documentation/index.rst

```diff
@@ -1,158 +1,152 @@
 Welcome to the FlexMeasures documentation!
 ===================================================================
 
-When we use a lot of renewable energy, flexibility is becoming crucial and valuable, e.g. for demand response.
 FlexMeasures is the intelligent & developer-friendly EMS to support real-time energy flexibility apps, rapidly and scalable.
 
-The problem it helps to solve is:
-
-*What are the best times to run flexible assets, like batteries or heat pumps?*
+The problem it helps you to solve is: **What are the best times to run flexible assets, such as batteries or heat pumps?**
 
 In a nutshell, FlexMeasures turns data into optimized schedules for flexible assets.
-Why? Planning ahead allows flexible assets to serve the whole system with their flexibility, e.g. by shifting energy consumption to other times.
-For the asset owners, this creates CO₂ savings but also monetary value (e.g. through self-consumption, dynamic tariffs and grid incentives).
+*Why?* Planning ahead allows flexible assets to serve the whole system with their flexibility, e.g. by shifting energy consumption to more optimal times. For the asset owners, this creates CO₂ savings but also monetary value (e.g. through self-consumption, dynamic tariffs and grid incentives).
 
 
 .. image:: https://raw.githubusercontent.com/FlexMeasures/screenshots/main/architecture/simple-flexEMS.png 
     :align: center
 ..    :scale: 40%
 
+FlexMeasures is written in Python, and runs on Flask and Postgres.
+We aim to create developer-friendly technology that saves time in developing complex services.
+Read more on this in :ref:`dev_why`.
 
-However, developing apps & services around energy flexibility is expensive work. FlexMeasures is designed to be developer-friendly, which helps you to go to market quickly, while keeping the costs of software development at bay. 
-FlexMeasures supports:
-
-- Real-time data integration & intelligence 
-- Model data well ― units, time resolution & uncertainty (of forecasts)
-- Faster app-building (API/UI/CLI, plugin & multi-tenancy support)
-
-More on this in :ref:`dev_tooling`. FlexMeasures proudly is an incubation project at `the Linux Energy Foundation <https://www.lfenergy.org/>`_. Also, read more on where FlexMeasures is useful in :ref:`use_cases`.
-
+FlexMeasures proudly is an incubation project at `the Linux Energy Foundation <https://www.lfenergy.org/>`_.
 
-A quick glance at usage
-------------------------
 
-A tiny, but complete example: Let's install FlexMeasures from scratch. Then, using only the terminal (FlexMeasures of course also has APIs for all of this), load hourly prices and optimize a 12h-schedule for a battery that is half full at the beginning. Finally, look at our new schedule.
+A quick glance 
+----------------
 
-.. code-block:: console
+.. tabs::
```

Comment:
The nested tabs don't really appeal to me. Would you consider one of these alternatives (which would still require adapting some of the accompanying texts)?

## Flattening the nesting of tabs

![Peek 2024-02-06 12-02 tabs](https://github.com/FlexMeasures/flexmeasures/assets/30658763/153e3cf0-f216-4ef7-9408-d2b3f5b07d30)

## Moving the code example to a dropdown admonition

Just let me know if you need the code for this.

![Peek 2024-02-06 12-02 admonition](https://github.com/FlexMeasures/flexmeasures/assets/30658763/c2c47660-a4a4-4fed-ab34-74055c9bd3b5)

Reply:
I like your first example and implemented it.